### ISSUE TITLE
Updated definition for change of context and relative luminance

### ISSUE THREAD

**Issue by kengdoj** [atbcb/ICTTestingBaseline#342]
https://github.com/w3c/wcag/pull/1765/files

"Change of context" is in multiple baselines. What impact does this definition change have?

**Comment by bruce-usab** [comment]
Just a few more data points:
- WCAG 2.0 errata page is here:  https://www.w3.org/WAI/WCAG20/errata/
- Definition change will not go live until 2.2 posts as TR (hopefully, June)
- This is not a substantive change
- Hopefully (I say) there will be other errata.  For example, the definition for [Relative Luminace](https://www.w3.org/TR/WCAG20/#relativeluminancedef) has a number wrong in the forth decimal point.

**Comment by kengdoj** [comment]
Thanks, @bruce-usab.

If the errata is available for the change of context definition, update the "Change of context" definition, referencing the errata, in two Baseline Tests: 
- 2 Focus for [WCAG SC 3.2.1 On Focus](http://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-receive-focus.html) – When any component receives focus, it does not initiate a **change of context.**
- 10 Forms for [WCAG 3.2.2 On Input](http://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-unpredictable-change.html): Changing the setting of any user interface component does not automatically cause a **change of context** unless the user has been advised of the behavior before using the component.

The relative luminance update in the errata should be noted in Baseline Test 9. Flashing, under Test Method Rationale: 
- Per [WCAG 2.0 errata](https://www.w3.org/WAI/WCAG20/errata/): in the [definition of relative luminance](https://www.w3.org/TR/WCAG20/#relativeluminancedef), the red threshold was updated from 0.03928 to 0.04045.

**Comment by kengdoj** [comment]
Discussed and decided: 
- update links to 2.2 version of relative luminance (draft: https://w3c.github.io/wcag/guidelines/22/#dfn-relative-luminance), 
- and 2.2's definition of change of context (draft: https://w3c.github.io/wcag/guidelines/22/#dfn-change-of-context).

**Comment by bruce-usab** [comment]
Thanks @kengdoj &mdash; I missed that the formal 2.0 errata document has been updated to the correct red threshold value (0.03928 to 0.04045).

**Comment by kengdoj** [comment]
Closed by #451
